.github/workflows/release.yml

```diff
@@ -5,6 +5,9 @@ on:
     tags:
       - "v*"
 
+permissions:
+  contents: write
```

Comment:
@Dunqing it was silently failing on Github actions, I think because of this missing permission.